src/components/MCQ/MCQQuiz.vue

```diff
@@ -3,34 +3,58 @@
     v-if="currentQuestion"
     :statement="currentQuestion.statement"
     :options-list="currentQuestion.optionsList"
+    :_id="currentQuestion._id"
     @next-question="nextQuestion"
     @skip-question="skipQuestion"
   />
-  <div v-if="!currentQuestion">You are done! Please refresh this page.</div>
+  <MCQStatus
+    v-if="!currentQuestion"
+    :quiz-status="questionsQueue.quizStats"
+    :work-quiz="questionsQueue.quizStats.length"
```

Comment:
probably can delete these props just use questionsQuene.quizStats directly in MCQStatus. in another code refactoring ticket or?

Reply:
This one should be part of a refactoring, I can only change anything minor at this level but once we start dealing with multiple changes such as props and types, it should be part of a ticket.